ooo-request/README.md

```diff
@@ -0,0 +1,87 @@
+### OOO Status request Firestore collection model
+
+```json
+{
+  "id": "String",
+  "userId": "String",
+  "state": "PENDING",
+  "from": "Timestamp",
+  "until": "Timestamp",
+  "message": "String",
+  "createdAt": "Timestamp",
+  "updatedAt": "Timestamp",
+  // after approval
+  "state": "APPROVED",
+  "approvedAt": "Timestamp",
+  "approvedBy": "String",
```

Comment:
- Won't the updatedAt and approvedAt time be same? 
- Why do we need separate fields?

Reply:
Yes both are the same then we can keep the `updatedAt` field only